lang/src/test/kotlin/org/partiql/lang/eval/builtins/functions/CharacterLengthEvaluationTest.kt

```diff
@@ -17,7 +18,7 @@ class CharacterLengthEvaluationTest : EvaluatorTestBase() {
     @ParameterizedTest
     @ArgumentsSource(PassCases::class)
     fun characterLengthPassTests(testCase: ExprFunctionTestCase) =
-        runEvaluatorTestCase(testCase.source, expectedResult = testCase.expectedLegacyModeResult)
+        runEvaluatorTestCase(testCase.source, expectedResult = testCase.expectedLegacyModeResult, expectedPermissiveModeResult = testCase.expectedPermissiveModeResult)
```

Comment:
Why is this change?

Reply:
`char_legnth(missing)` outputs `null` in legacy typing mode, and outputs `missing` in permissive mode. Originally, since we were using `ION_WITHOUT_ANNOTATION`, `null` in ION can represent both `missing` & `null`. But now we need to change.

Reply:
The `expectedPermissiveModeResult` is the same as `expectedResult`, if not specified.

Reply:
I see now: this goes in conjunction of adding a permissive-result to the tests cases where it differs from the legacy-result, as on Line 33 below.  

This is a kind of thing worth mentioning in the PR description as a finding.